packages/ui-components/src/components/Map/utils/markerFormats.js

```diff
@@ -274,26 +310,10 @@ export function getMeasureDisplayInfo(
       displayInfo.radius = radius;
     }
   });
-  series.forEach(
-    ({
-      key,
-      type,
-      valueMapping,
-      noDataColour,
-      scaleType,
-      scaleColorScheme,
-      min,
-      max,
-      hideByDefault,
-    }) => {
-      const valueInfo = getValueInfo(measureData[key], valueMapping, {
-        ...hideByDefault,
-        ...hiddenValues[key],
-      });
+  serieses.forEach(
+    ({ key, type, valueMapping, noDataColour, scaleType, scaleColorScheme, min, max }) => {
+      const valueInfo = getValueInfo(measureData[key], valueMapping);
 
-      if (valueInfo.isHidden) {
-        displayInfo.isHidden = true;
-      }
       displayInfo.originalValue = measureData.originalValue;
```

Comment:
It seems like this `originalValue` is a bit dodgy too...it'll be the value of the final series. We don't currently use it in any meaningful way in `ui-components` or `lesmis` though, so it's probably fine until we integrate with tupaia.org